Jenkinsfile

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env groovy
+
+library("govuk")
```

Comment:
govuk-jenkinslib assumes the default branch is `master`, use this branch of it instead:

```
library("govuk@default-branch")
```

Reply:
ace thanks very much @barrucadu !